e2etests/test/twin_e2e_tests.js

```diff
@@ -171,7 +171,7 @@ delete nullMergeResult.tweedle;
       compare(right, left);
     };
 
-    it('relies on $version starting at 1 and incrementing by 1 each time', function(done) {
+    it.skip('relies on $version starting at 1 and incrementing by 1 each time', function(done) {
```

Comment:
>it.skip('relies on $version starting at 1 and incrementing by 1 each time', function(done) { [](start = 4, length = 92)

I added a task to the board under build issues to investigate this.

Reply:
thanks!